utils/build/docker/nodejs/express4/integrations/db/postgres.js

```diff
@@ -1,87 +1,85 @@
-"use strict";
+'use strict'
 
 const { join } = require('path')
-const { Client, Pool } = require('pg')
-const { readFileSync, statSync } = require('fs')
+const { Client } = require('pg')
+const { readFileSync } = require('fs')
 
-async function launchQuery(query) {
-    return new Promise(function (resolve, reject) {
-        const client = new Client()
-        client.connect().then(() => {
-            client.query(query).then((queryResult) => {
-                console.log("Postgres query result:" + queryResult);
-                resolve("Postgres query done!")
-            }).catch((err) => {
-                console.log(err)
-                resolve("Error on postgres query")
-            })
-        }).catch((err) => {
-            console.log(err)
-            reject("Error on postgres connection")
-        })
-    });
+async function launchQuery (query) {
+  return new Promise(function (resolve, reject) {
+    const client = new Client()
+    client.connect().then(() => {
+      client.query(query).then((queryResult) => {
+        console.log('Postgres query result:' + queryResult)
+        resolve('Postgres query done!')
+      }).catch((err) => {
+        console.log(err)
+        resolve('Error on postgres query')
+      })
+    }).catch((err) => {
+      console.log(err)
+      reject(err)
+    })
+  })
 }
 
-async function initData() {
-    console.log("loading sql data")
-    const query = readFileSync(join(__dirname, 'resources', 'postgres.sql')).toString()
-    return await launchQuery(query);
+async function initData () {
+  console.log('loading sql data')
+  const query = readFileSync(join(__dirname, 'resources', 'postgres.sql')).toString()
+  return await launchQuery(query)
 }
 
-async function select() {
-    const sql = "SELECT * FROM demo where id=1 or id IN (3, 4)"
-    const client = new Client()
```

Comment:
potential side effect here

Reply:
I don't think so. 
The rest of the methods do not create an instance and tests are passing without it.

Reply:
true, i'll let the APM team decide